src/core/mac/mac.cpp

```diff
@@ -991,8 +991,7 @@ void Mac::ProcessTransmitSecurity(TxFrame &aFrame, bool aProcessAesCcm)
 
         break;
 
-    case Frame::kKeyIdMode2:
-    {
+    case Frame::kKeyIdMode2: {
```

Comment:
nit: why change the style?
may run `./script/make-pretty` to format the code style?